source/clear-linux/tutorials/nvidia-cuda.rst

```diff
@@ -0,0 +1,275 @@
+   development environment, including the host compiler and C runtime
+   libraries, and is therefore only supported on distribution versions that
+   have been qualified for this CUDA Toolkit release. 
+
+Refer to the `NVIDIA documentation on CUDA system requirements
+<https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#system-requirements>`_
+for the latest kernel and compiler compatibility. 
+
+For example, CUDA 10.1 on a system with the latest Linux kernel requires GCC7,
+which is older than the default GCC version for |CL|.
+
+Install the compatible version of GCC, if required:
+
+#. Install the :ref:`bundle <bundles>` with the appropriate GCC version.
+
+   .. code:: bash
+
+      sudo swupd bundle-add c-extras-gcc7
+      
+#. Create the directory :file:`/usr/local/cuda/bin`:
+
+   .. code:: bash
+
+      sudo mkdir -p /usr/local/cuda/bin
+
+
+#. Add symlinks to the older GCC version in the
+   :file:`/usr/local/cuda/bin` directory. This will cause the older version of
+   GCC to be used when :file:`/usr/local/cuda/bin` is in the $PATH environment
+   variable.
+
+   .. code:: bash
+   
+      sudo ln -s /usr/bin/gcc7 /usr/local/cuda/bin/gcc
+      sudo ln -s /usr/bin/g++7 /usr/local/cuda/bin/g++  
+
+
+Downloading and Installation
+****************************
+
+
+Download the NVIDIA CUDA Toolkit
+================================
+
+#. Go to the `NVIDIA CUDA downloads website`_ to get the latest CUDA Toolkit. 
+   If an older version of the CUDA Toolkit is required, go to the `CUDA
+   Toolkit Archive <https://developer.nvidia.com/cuda-toolkit-archive>`_.
+
+   Choose the following settings and click *Download*. 
+   
+   - Operating System: *Linux*
+   - Architecture: *x86_64*
+   - Distribution: *any*
+   - Version: *any*
+   - Installer Type: *runfile(local)*
+
+
+
+#. Open a terminal and navigate to where the
+   :file:`cuda_<VERSION>_linux.run` file was saved. In this
+   example, it was saved in the Downloads folder.
+
+   .. code-block:: bash
+
+      cd ~/Downloads/
+
+#. Make the :file:`cuda_<VERSION>_linux.run` file executable:
+
+   .. code-block:: bash
+
+      chmod +x cuda_<VERSION>_linux.run
+
+
+
+Install the NVIDIA CUDA Toolkit
+===============================
+
+The NVIDIA CUDA installer will be directed to install files under
+:file:`/opt/cuda` as much as possible to keep its contents isolated from the
+rest of the |CL| files under :file:`/usr`. 
+
+The CUDA installer automatically creates a symbolic link that allows the CUDA
+Toolkit to be accessed from :file:`/usr/local/cuda` regardless of where it was
+installed.
+
+
+#. Configure the dynamic linker to look for and cache shared libraries under
+   :file:`/opt/cuda/lib64` where the NVIDIA installer will place libraries. 
+
+   .. code-block:: bash
+      
+      echo "include /etc/ld.so.conf.d/*.conf" |  sudo tee --append /etc/ld.so.conf
+      sudo mkdir /etc/ld.so.conf.d
```

Comment:
I'd swap these two lines (do the mkdir first) as a matter of pragmatism, plus add "-p" to mkdir.